docs/source/tutorials/hospital_admissions_model.qmd

```diff
@@ -535,21 +540,29 @@ And plotting the results:
 out = hosp_model_weekday.plot_posterior(
     var="latent_hospital_admissions",
     ylab="Hospital Admissions",
-    obs_signal=np.pad(dat_w_padding, (gen_int_array.size, 0), constant_values=np.nan),
+    obs_signal=np.pad(
+        daily_hosp_admits.astype(float),
+        (gen_int_array.size + pad_size, 0),
+        constant_values=np.nan,
+    ),
 )
 ```
 
 We will use ArviZ to visualize the posterior/prior predictive distributions.
+By increasing `n_timepoints_to_simulate`, we can perform forecasting.
```

Comment:
Potentially a bit confusing to have this line here but then go into prior predictive analysis and come back to forecasting further down, without mentioning that that's what it is. Perhaps insert something highlighting forecasting here?

https://github.com/CDCgov/multisignal-epi-inference/blob/1795426650682eab3ab7367e20ea8a5365a5d5e4/docs/source/tutorials/hospital_admissions_model.qmd#L602

Reply:
addressed in https://github.com/CDCgov/multisignal-epi-inference/pull/241/commits/a84ca20e4e1183033df3c65e0c3b061ad2f410e7